components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/impl/ServerSessionManagementServiceImpl.java

```diff
@@ -94,13 +97,26 @@ private void terminateSession(SessionContext sessionContext, String sessionId) {
     private void addAuditLogs(String sessionKey, String initiator, String authenticatedUser, String userTenantDomain,
                               String traceId, Long terminatedTimestamp) {
 
+        String initiatedUser = null;
```

Comment:
why do we need to define a new variable? can't we directly use initiator?

Reply:
If we cannot find the userId of the initiator(when userId is null) when the log masking is enabled, the username will get masked directly. That's the reason to define a new variable to keep the initiator userId instead of modifying the initiator variable.